py3status/module.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, module, user_modules, py3_wrapper):
         self.prevent_refresh = False
         self.sleeping = False
         self.timer = None
+        self.urgent = False
```

Comment:
The reason for the class parameter is that we want to know when a module first becomes urgent so we can switch the group to show it.  The user may then change the group to show something else.  If we didn't have this then the group would keep showing the urgent module each time it was updated.